admin/src/components/Login/Login.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class SignUp extends React.PureComponent {
+  state = {
+    isLoading: false,
+    name: null,
+    email: null,
+    password: null
+  };
+
+  onSignUp(e) {
```

Comment:
this component is called login but calls register and refers to everything as sign up?

Reply:
Since I created the test in another branch, with the setup of jest also, will remove this for now.